src/requestManager.cpp

```diff
@@ -286,7 +286,7 @@ CNodeRequestData::CNodeRequestData(CNode* n)
   desirability -= latency;
 }
 
-void CUnknownObj::AddSource(CNode* from)
+bool CUnknownObj::AddSource(CNode* from)
 {
   if (std::find_if(availableFrom.begin(), availableFrom.end(), IsCNodeRequestDataThisNode(from)) == availableFrom.end())  // node is not in the request list
```

Comment:
once the std:: is prefixed it becomes even longer.
I don't want to fix the braces as part of this PR.  Looks like the "janitor" is needed here as well as in the dev branch